variables.tf

```diff
@@ -110,6 +110,16 @@ variable "mirror_repositories" {
       description = "Mage-OS mirror fork of https://github.com/magento/zf1."
       teams       = ["continuous-integration"]
     }
+
+    mirror-magento-cloud-patches = {
+      description = "This is a Mage-OS fork of the Magento Cloud Patches project found at https://github.com/magento/magento-cloud-patches."
+      teams       = ["infrastructure"]
```

Comment:
All current mirror repositories have `continuous-integration` as a team. Why do the new ones have `infrastructure`?

Reply:
You're right. We decided it during the call, but we likely made a mistake; I will correct it.